.circleci/config.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+jobs:
+  build:
+    docker:
+    machine:
```

Comment:
Is 'docker:' correct? Should this be 'docker: {}'?

Reply:
stray key, fixed!